share/functions/cd.fish

```diff
@@ -3,10 +3,13 @@
 #
 function cd --description "Change directory"
     set -l MAX_DIR_HIST 25
+    set -l options h/help P
+    argparse -n cd --max-args=2 $options -- $argv
+    or return
 
-    if test (count $argv) -gt (test "$argv[1]" = "--" && echo 2 || echo 1)
-        printf "%s\n" (_ "Too many args for cd command")
-        return 1
+    # Drop the -P argument as fish always resolves the directory
```

Comment:
This hasn't been true in a while - since 3.0.

Fish's default is `cd -L`, like POSIX.

Reply:
Sorry for that, you're right. I have fixed that.